### DISCUSSION TITLE
[Tutorial] Cómo actualizar un repositorio "Forkeado"

### DISCUSSION THREAD

**Discussion by DanielOe21** [LaunchX-InnovaccionVirtual/CursoIntroPython#196] [Show and tell]
# Actualiza el repositorio que ´Forkeaste´

Como ya te debiste haber dado cuenta, ya se agregaron las últimas Katas al repositorio del curso, **pero** si tú hiciste un FORK desde antes, en tu perfil no se actualiza automáticamente.

### El repositorio ´original´ se ve así:
![image](https://user-images.githubusercontent.com/94386100/153720862-40282daf-1905-4b95-82f1-d82de0274bc4.png)

### El tuyo puede verse así:
![image](https://user-images.githubusercontent.com/94386100/153720953-74a467a0-f1bc-4044-8bc2-d3d3ac9b1c26.png)

### 1. Revisa tu estatus

Para actualizar tu repositorio con los nuevos cambios, abre TU repositorio y observa en la parte de arriba de los archivos, verás una leyenda parecida a la de la imagen, quiere decir que hay diferencias entre la copia que tu tienes y el repositorio original, da clic en ¨Fetch upstream¨:
![image](https://user-images.githubusercontent.com/94386100/153721059-d9067461-00fb-4044-977c-788b14a2e1cf.png)

### 2. Pide la actualización

Da clic en el botón verde
![image](https://user-images.githubusercontent.com/94386100/153721230-44dfa5dd-8f14-4019-b119-c2446b90acab.png)

### 3. Listo! Ya estás al día! 
![image](https://user-images.githubusercontent.com/94386100/153721258-99be725e-6cd3-4101-9e51-90ecc17b12c1.png)

NOTA: Este proceso es para ´Actualizar´ tu copia del repositorio, como las carpetas de las primeras katas no tuvieron cambios, sólo se agregarán las katas nuevas, así que si ya tienes las primeras resueltas **no se verán afectadas**

## Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>

**Comment by Laura-Karina-GR** [comment]
Gracias

**Reply by DanielOe21** [reply]
Un gusto!

**Comment by DiegoPenalozaJuarez** [comment]
Crack!! Gracias por el consejo!!

De mucha ayuda! 💯

**Reply by DanielOe21** [reply]
Estamos para apoyarnos!

**Comment by venegas55** [comment]
Excelente, solo una pregunta:

exáctamente como dices, al no haberse actualizado las katas que ya resolvimos el "upstream" va solito...

pero ¿qué pasa si existen cambios en las katas? ¿se sobreescribe o te menciona los cambios y tu ves que aceptas o no del upstream?

**Reply by DanielOe21** [reply]
Excelente (y temida) pregunta, la verdad no soy para nada un experto de GitHub, de hecho justo en este curso es donde estoy aprendiendo a usarlo 😅 Pero por intuición y leyendo un poco de info en internet, supongo que si le das clic en ' compare'  te aparecerán las diferencias entre el repo forkeado y el original, así que antes de aceptar el Merge, podrías en teoría ver que cambios aceptas y cuales rechazas... Pero es pura deducción propia 😥 el día que lo confirme te aviso!

**Reply by venegas55** [reply]
va va, voy a hacer un "ejercicio" forkeando a otra cuenta mía y haciendo cambios en los cambios xD

**Reply by DanielOe21** [reply]
me avisas como te va!!!
